odk/odk.py

```diff
@@ -150,6 +150,9 @@ class ImportProduct(Product):
     use_gzipped: bool = False
     """if use_gzipped is true, try use the base IRI instead of normal one to mirror from."""
 
+    mirror_type: Optional[str] = None
```

Comment:
is `None` here not something else then the string `'none'` you look for later?

Reply:
No. It's the meaning in the python. It's an optional string value, then the default is None. Same as `module_type`.

Reply:
Ok